lms/services/canvas_api/_authenticated.py

```diff
@@ -58,23 +54,10 @@ def send(
             access token for the user is missing, expired, or has been deleted
         """
         call_args = (method, path, schema, timeout, params)
```

Comment:
Packing the `call_args` here was useful before, but I think it's a bit weird on it's own now. This can be folded into the `send()` call now I think

Reply:
Done